docs/manual/developer_guide.adoc

```diff
@@ -399,14 +399,15 @@ make vendor-zipfile
 make redhat-zipfile
 ```
 
-### Build the docker container image
+### Build the Docker container image
 
-Use a suitable Dockerfile present in the
+Find a suitable Dockerfile present in the
 link:https://github.com/ComplianceAsCode/content/tree/master/Dockerfiles[Dockerfiles]
-directory and build the image.
+directory, copy the file to the top directory and build the image.
 This will take care of the build environment and all necessary setup.
 
 ```bash
+cp Dockerimages/ubuntu ./Dockerfile
```

Comment:
Instead of copying the Dockerfile you can use `--file Dockerfiles/ubuntu` in the `docker` command. Note also that the directory name is `Dockerfiles`, not `Dockerimages`.

Reply:
Thanks for catching that. Fixed.